build/conda/README.md

```diff
@@ -0,0 +1,28 @@
+# OpenBB Platform Installer
+
+The code in this folder allows creating a packaged installer for the OpenBB Platform using conda constructor.
+
+The installers can be created for Windows (.exe), MacOS (.pkg) and Linux (.sh).
```

Comment:
For Linux users, they need to have other things installed before attempting to install `openbb[all]`, like the Docker file - [https://github.com/OpenBB-finance/OpenBB/blob/develop/build/docker/platform.dockerfile](https://github.com/OpenBB-finance/OpenBB/blob/develop/build/docker/platform.dockerfile)

Reply:
Potentially overcome this with: https://github.com/OpenBB-finance/OpenBB/pull/6590